src/py/_pyodide/_core_docs.py

```diff
@@ -247,7 +247,7 @@ def to_file(self, file: IOBase, /):
         data once.
         """
 
-    def from_file(self, file: IOBase, /):
+    def from_file(self, file: IOBase, /) -> Any:
```

Comment:
https://github.com/pyodide/pyodide/blob/af972267c7969c6f023da709851463ec99e7e4ac/src/core/jsproxy.c#L1743
```suggestion
    def from_file(self, file: IOBase, /) -> None:
```